projects/objectnav_baselines/experiments/objectnav_mixin_ddppo.py

```diff
@@ -14,7 +35,29 @@
 
 
 class ObjectNavMixInPPOConfig(ObjectNavBaseConfig):
+    # selected auxiliary uuids
+    ## if comment all the keys, then it's vanilla DD-PPO
+    AUXILIARY_UUIDS = [
+        # InverseDynamicsLoss.UUID,
+        # TemporalDistanceLoss.UUID,
+        # CPCA1Loss.UUID,
+        # CPCA4Loss.UUID,
+        # CPCA8Loss.UUID,
+        # CPCA16Loss.UUID,
+    ]
+
+    add_prev_actions = False
+
+    multiple_beliefs = False
+    beliefs_fusion = (  # choose one
+        None
+        # AttentiveFusion.UUID
+        # AverageFusion.UUID
+        # SoftmaxFusion.UUID
+    )
```

Comment:
As these are constants, can we make them fully capitalized (i.e. like `AUXILIARY_UUIDS`)?